### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Name property is empty for button in "Create Authentication Files dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1386]
**Repro Steps:**

1. Open and Start Eclipse.
2. Navigate to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to and select "Automated" radio button.
4. Tab to and select "Browse" button.
5. In "Create Authentication Files" dialog , tab to button and observe the name property using inspect.

**Actual Result:**
Name property is empty for the button

**Narrator behaviour:**
button

**NVDA behaviour:**
button

**Expected Result:**
Name property for the button should be "Browse".

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![create authentication file-button-no name](https://user-images.githubusercontent.com/22956130/35681878-da981034-0784-11e8-843f-0848950de674.jpg)
![create authentication file-button-no name-nvda](https://user-images.githubusercontent.com/22956130/35681882-ddc0c1e8-0784-11e8-955a-a3ff4185df02.jpg)

**Comment by Eskibear** [comment]
shipped in v3.6.0

**Comment by WiproPSAccTest** [comment]
Verified the issue in V.3.6.0 .Issue got fixed.
Please add closed tag.
![1386-fixed](https://user-images.githubusercontent.com/22956130/39762242-c249ac04-52f7-11e8-837d-12199fb05959.jpg)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749851
